yt_dlp/extractor/pinterest.py

```diff
@@ -24,14 +24,51 @@ def _call_api(self, resource, video_id, options):
 
     def _extract_video(self, data, extract_formats=True):
         video_id = data['id']
+        thumbnails = []
+        images = data.get('images')
+        if isinstance(images, dict):
+            for thumbnail_id, thumbnail in images.items():
+                if not isinstance(thumbnail, dict):
+                    continue
+                thumbnail_url = url_or_none(thumbnail.get('url'))
+                if not thumbnail_url:
+                    continue
+                thumbnails.append({
+                    'url': thumbnail_url,
+                    'width': int_or_none(thumbnail.get('width')),
+                    'height': int_or_none(thumbnail.get('height')),
+                })
 
-        title = (data.get('title') or data.get('grid_title') or video_id).strip()
+        info = {
+            'id': video_id,
+            'title': (data.get('title') or data.get('grid_title') or video_id).strip(),
+            'description': data.get('seo_description') or data.get('description'),
+            'timestamp': unified_timestamp(data.get('created_at')),
+            'thumbnails': thumbnails,
+            'uploader': traverse_obj(data, ('closeup_attribution', 'full_name')),
+            'uploader_id': str_or_none(traverse_obj(data, ('closeup_attribution', 'id'))),
+            'repost_count': int_or_none(data.get('repin_count')),
+            'comment_count': int_or_none(data.get('comment_count')),
+            'categories': traverse_obj(data, ('pin_join', 'visual_annotation'), expected_type=list),
+            'tags': traverse_obj(data, 'hashtags', expected_type=list),
+            'extractor_key': PinterestIE.ie_key(),
```

Comment:
```suggestion
            'extractor_key': PinterestIE.ie_key(),
            'extractor': PinterestIE.IE_NAME,
            'webpage_url': ...,
```
Could u also add these?